hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.test.js

```diff
@@ -77,14 +79,25 @@ describe("<UnconnectedCheckedOutTable />", () => {
 
     it("Hides the table when isVisible is false", () => {
         const { getByText, queryByText } = render(
-            <UnconnectedCheckedOutTable items={itemsCheckedOut} isVisible={false} />
+            withRouter(
+                <UnconnectedCheckedOutTable items={itemsCheckedOut} isVisible={false} />
+            )
         );
         expect(getByText(/checked out items/i)).toBeInTheDocument();
         expect(getByText(/show all/i)).toBeInTheDocument();
         itemsCheckedOut.map(({ name }) => {
             expect(queryByText(name)).toBeNull();
         });
     });
+    it("Goes to the incident form when you click the 'Report broken/lost' button", () => {
```

Comment:
The test does not follow from this description.

Reply:
oops sorry forgot to change that